ansible/roles/rancher/tasks/cert-manager.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Probe cert-manager Custom Resource Definition
+  command: kubectl get CustomResourceDefinition certificates.certmanager.k8s.io
+  run_once: true
+  ignore_errors: true
+  changed_when: false
+  register: crd
+  
+- name: Install cert-manager Custom Resource Definition
+  command: kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.9/deploy/manifests/00-crds.yaml
+  run_once: true
+  when: crd is failed
+
+- name: Probe cert-manager Namespace
+  command: kubectl get ns cert-manager
+  changed_when: false
+  ignore_errors: yes
+  run_once: yes
+  register: certmanager_ns
+
+- name: Create cert-manager Namespace
+  command: kubectl create namespace cert-manager
+  run_once: true
+  when: certmanager_ns is failed
+
+- name: Probe cert-manager Namespace Label
+  command: kubectl get namespace --selector certmanager.k8s.io/disable-validation=true
+  changed_when: false
+  run_once: true
+  register: certmanager_ns_label
+
+- name: Disable Resource Validation on cert-manager Namespace
+  command: kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
+  run_once: true
+  when: "'No resources found' in certmanager_ns_label.stderr"
+
+- name: Add Jetstack Repo for cert-manager
+  command: helm repo add jetstack https://charts.jetstack.io
+  when: "'jetstack' not in repos.stdout"
+  run_once: true
+  register: jetstack_repo
+
+- name: Update Helm Repo
+  command: helm repo update
+  when: jetstack_repo is changed
+
+- name: Install cert-manager
+  command: helm install --name cert-manager --namespace cert-manager --version v0.9.1 jetstack/cert-manager --description='RanchHand Deploy' --wait
+  when: "'cert-manager' not in charts.stdout"
```

Comment:
I did, but I couldn't get it to work! I think it would just take a full day to get working but I had spend a few hours with no success. I have this on my mental backlog to update this and use the native `helm` module (same for k8s). I can also create a real task for this to track. If I have some time, I can play around with this. I typically hate using `command` module hacks.